assembl/static2/css/components/fictionPreview.scss

```diff
@@ -0,0 +1,117 @@
+.fiction-preview {
+  border-bottom: solid 2px $theme-local-blue;
+  box-shadow: 0 2px 3px 0 #d0d0d0;
+  width: 149px;
+  height: 200px;
+  margin: 4px;
+
+  &:hover {
+    transition: all 0.3s ease-out;
+    transform: scale(1.02);
+  }
+
+  .light-title-3,
+  p,
+  .author,
+  .published-date {
+    font-family: 'LatoRegular';
+    font-weight: normal;
```

Comment:
@cedrtang is 
    font-weight: normal;
    font-style: normal;
    font-stretch: normal;
    letter-spacing: normal;
    text-transform: none;
    text-align: left;
required ? Seems if styling is not different from the inherited style, should be deleted ?